ptbcontrib/username_to_chat_api/README.md

```diff
@@ -40,10 +40,10 @@ logging.basicConfig(
 class CustomContext(CallbackContext):
     def __init__(self, application: Application):
         super().__init__(application=application)
```

Comment:
no need for init now :) yes, I'm pick 🙆‍♂️

Reply:
Grmpf you could have said that earlier

Reply:
though it was obvious 😬